<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Unit_setting_dialog</class>
 <widget class="QDialog" name="Unit_setting_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>λ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>nm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>μm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mm</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="prefix">
    <string>1λ = </string>
   </property>
   <property name="suffix">
    <string> nm</string>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="sureBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sure</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
